packages/mdc-dialog/mdc-dialog.scss

```diff
@@ -40,148 +40,148 @@ $mdc-dialog-dark-theme-bg-color: #303030 !default;
+  flex-direction: column;
+  width: calc(100% - 30px);
+  min-width: 640px;
+  max-width: 865px;
+  transform: translateY(150px) scale(.8);
+  border-radius: 2px;
+  opacity: 0;
 
-    @include mdc-rtl(".mdc-dialog") {
-      text-align: right;
-    }
+  @include mdc-theme-dark(".mdc-dialog") {
+    @include mdc-theme-prop(color, text-primary-on-dark);
 
-    &__title {
-      @include mdc-typography(title);
+    background-color: $mdc-dialog-dark-theme-bg-color;
+    background-color: var(--mdc-dialog-dark-theme-bg-color, #{$mdc-dialog-dark-theme-bg-color});
+  }
 
-      flex: 1;
-      margin: 0;
-    }
+  @include mdc-rtl(".mdc-dialog") {
+    text-align: right;
   }
+}
+
+.mdc-dialog__header {
+  display: flex;
+  align-items: center;
+  padding: 24px 24px 0;
 
-  &__body {
-    @include mdc-theme-prop(color, text-secondary-on-light);
-    @include mdc-typography(body1);
-
-    margin-top: 20px;
-    padding: 0 24px 24px;
-
-    @include mdc-theme-dark(".mdc-dialog") {
-      @include mdc-theme-prop(color, text-secondary-on-dark);
-    }
-
-    &--scrollable {
-      max-height: 195px;
-      border-top: 1px solid rgba(0, 0, 0, .1);
-      border-bottom: 1px solid rgba(0, 0, 0, .1);
-      overflow-x: auto;
-      overflow-y: scroll;
-      -webkit-overflow-scrolling: touch;
-    }
+  @include mdc-rtl(".mdc-dialog") {
+    text-align: right;
   }
+}
+
+.mdc-dialog__header__empty {
+  padding: 0;
+}
 
-  &__footer {
-    display: flex;
-    flex-wrap: wrap;
-    align-items: center;
-    justify-content: flex-end;
-    padding: 8px;
+.mdc-dialog__header__title {
+  @include mdc-typography(title);
+
+  flex: 1;
+  margin: 0;
+}
 
-    &__button {
-      @include mdc-rtl-reflexive-box(margin, right, 8px);
+.mdc-dialog__body {
+  @include mdc-theme-prop(color, text-secondary-on-light);
+  @include mdc-typography(body1);
 
-      &:last-child {
-        @include mdc-rtl-reflexive-box(margin, right, 0);
-      }
-    }
+  margin-top: 20px;
+  padding: 0 24px 24px;
+
+  @include mdc-theme-dark(".mdc-dialog") {
+    @include mdc-theme-prop(color, text-secondary-on-dark);
   }
+}
+
+.mdc-dialog__body--scrollable {
+  max-height: 195px;
+  border-top: 1px solid rgba(0, 0, 0, .1);
+  border-bottom: 1px solid rgba(0, 0, 0, .1);
+  overflow-x: auto;
+  overflow-y: scroll;
+  -webkit-overflow-scrolling: touch;
+}
+
+.mdc-dialog__footer {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-end;
+  padding: 8px;
+}
 
-  &__action {
-    @include mdc-theme-prop(color, secondary);
+.mdc-dialog__footer__button {
+  @include mdc-rtl-reflexive-box(margin, right, 8px);
+
+  &:last-child {
+    @include mdc-rtl-reflexive-box(margin, right, 0);
   }
+}
 
-  // TODO: Replace with breakpoint variable
-  @media (max-width: 640px) {
-    min-width: 280px;
+.mdc-dialog__action {
+  @include mdc-theme-prop(color, secondary);
+}
+
+// TODO: Replace with breakpoint variable
+@media (max-width: 640px) {
+  min-width: 280px;
```

Comment:
This line should in a `.mdc-dialog` selector. 

The output css changed from 
```
  @media (max-width: 640px) {
    .mdc-dialog {
      min-width: 280px; }
      .mdc-dialog__surface { 
```
to 
```
@media (max-width: 640px) {
  min-width: 280px;
  .mdc-dialog__surface {
```

Reply:
Nice catch. Fixed.